File: RLE.API/Models/EnrolmentCenter.cs
using System;
using System.Collections.Generic;

namespace RLE.API.Models
{
    public partial class EnrolmentCenter
    {
        public EnrolmentCenter()
        {
            // Ecdeployment = new HashSet<Ecdeployment>();
            // Employee = new HashSet<User>();
            // InventOp = new HashSet<InventOp>();
            // SdcardTablet = new HashSet<SdcardTablet>();
        }

        public int Id { get; set; }
        public int? MunicipalityId { get; set; }
        public string Name { get; set; }
        public string Code { get; set; }

        public  Municipality Municipality { get; set; }
        // public  ICollection<Ecdeployment> Ecdeployment { get; set; }
        // public  ICollection<User> Employee { get; set; }
        // public  ICollection<InventOp> InventOp { get; set; }
        // public  ICollection<SdcardTablet> SdcardTablet { get; set; }
    }
}

File: RLE-SPA/src/app/_models/enrolmentCenter.ts
import { Municipality } from './municipality';
import { User } from './user';

export class EnrolmentCenter {
  public id: number;

  public municipalityId: number;

  public name: string;

  public code: string;

  public displayCode: string;

  public municipality: Municipality;
  public operators: User[];

  constructor(
    id?: number,
    municipalityId?: number,
    name?: string,
    code?: string,
    displayCode?: string,
    municipality?: Municipality,
  ) {
    this.id = id || 0;
    this.municipalityId = municipalityId || null;
    this.name = name || '';
    this.code = code || '';
    this.displayCode = displayCode || '';
  }
}
